wurst/event_handler/ChatHandler.wurst

```diff
@@ -201,8 +201,10 @@ init
                     stp.p.displayStats("|cffffcc00Average Time Alive as Sheep (-stimes / -sp)|r", p -> (p.timeAsSheepLiving / max(p.sheepPlayed, 1)).toTimeString(false))
                 case "-avgftimes"
                     stp.p.displayStats("|cffffcc00Average Time Before First Death as Sheep (-ftimes / -sp)|r", p -> (p.timeAsSheepFirst / max(p.sheepPlayed, 1)).toTimeString(false))
+                case "-d"
```

Comment:
Should toss this into the sheep only section, and I've been writing game messages starting with a verb and omitting pronouns, e.g. "Destroying all farms", and starting non-stat messages with SPACE_GOLD.

Thanks for contributing!

Reply:
Done